app/models/project_user_resource.rb

```diff
@@ -1,3 +1,6 @@
 class ProjectUserResource < ApplicationRecord
   belongs_to :project_user
+
+  validates :project_user_id, :percentage, :month, :year, :man_month,
```

Comment:
```suggestion
  validates :percentage, :month, :year, :man_month,
```
ko cần project_user_id nha, vì có validate tại line belongs_to project_user rồi